src/pages/ProjectDetails/Rewards/RewardForm/WorkItemSidePanel/OtherWorkForm/index.tsx

```diff
@@ -100,10 +100,10 @@ export default function OtherWorkForm({ projectId, contributorHandle, addWorkIte
   useMutationAlert({
     mutation: restcreateOtherWorkMutation,
     success: {
-      message: T("reward.form.contributions.other.success"),
+      message: T("reward.form.contributions.other.success", { item: "Other work" }),
```

Comment:
We may need to translate these types in the future, you could pass another translation as a parameter

```suggestion
      message: T("reward.form.contributions.other.success", { item: T('contributions.other') }),
```